WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -396,11 +395,8 @@ - (void)viewDidLoad
         AccountService *acctService = [[AccountService alloc] initWithCoreDataStack:[ContextManager sharedInstance]];
         [acctService updateUserDetailsForAccount:self.blog.account success:nil failure:nil];
     }
-    
-    [[NSNotificationCenter defaultCenter] addObserver:self
-                                             selector:@selector(handleDataModelChange:)
-                                                 name:NSManagedObjectContextObjectsDidChangeNotification
-                                               object:context];
+
+    [self observeManagedObjectContextObjectsDidChangeNotification];
```

Comment:
See this [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/21167/files#r1270643549) for more details.